### ISSUE TITLE
Skills

### ISSUE THREAD

**Issue by LilDocky** [shyzus/Legacy#16]
Human Skills
---Common---
Blast
Charge Blast
Shield
Solar Flare
Energy Manipulation
Zanzoken
Telepathy
Sense
Planet Sense
Destructo Disk
---Rare Skills---
Fly
Tribeam (Tien)
Mafuba (Roshi)
Burning Attack (Trunks)
Kamikaze Ghost Attack (Gotenks)
Spirit Ball (Yamcha)
---Super Rare Skills---
4 Arms (Tien)

Saiyan Skills
---Common---
Blast
Fly
Charge Blast
Shield
Energy Manipulation
Zanzoken
Oozaru Control
---Rare Skills---
Big Bang Attack (Vegeta)
Kamikaze Ghost Attack (Gotenks)
Burning Attack (Trunks)
Emelee: Ki Sword (Vegitto, Goku Black) (Sharp Emelee but extended hit range)
---Super Rare Skills---
Create Fake Moon

Namekian Skills
---Common---
Blast
Fly
Charge Blast
Shield
Energy Manipulation
Zanzoken
Sense
Planet Sense
Namekian Fusion
Namekian Fisson
Arm Stretch
==Fighter==
Regenerate
==Healer==
Heal
---Rare Skills---
Materialization (Piccolo, Kami)
Telepathy
Giant Form (Piccolo)
Hellzone Grenade (Piccolo)
---Super Rare Skills---
Sense Galaxy

Changelings
---Common---
Blast
Charge Blast
Shield
Energy Manipulation
Zanzoken
Transform
Revert
Telekinesis
---Rare Skills---
Death Ball (Freiza)

Makyojin
---Common---
Blast
Charge Blast
Shield
Energy Manipulation
Zanzoken
Telepathy
Sense
---Rare Skills---
Summon Deadzone Vortex
nigga i dont know I didn't watch the Garlic Jr. Saga

Demons
---Common---
Blast
Charge Blast
Shield
Energy Manipulation
Zanzoken
Telepathy
Destructo Disk
Decline Absorb
Demonic Teleport
Demon Revival
==Devil==
Soul Contract
==Demon==
Body Expand
---Rare Skills---
Soul Absorb
Petrify (Dabura)
Materialize Weapon (Dabura)

**Comment by Trazz666** [comment]
I know I mentioned it in the discord, but I'm adding here to make sure it's recorded where relevant. Soul contract is an existing skill for mages.  If I remember right, tricksters were meant to be the energy users of the demons, so perhaps giving them an energy skill instead is fitting.

**Comment by Matokushin** [comment]
The only thing I can see that I don't agree with right now is races restricted to emelee customization, which is unfair. In DBZ I'm pretty sure anyone could form a weapon with energy, use the assassin's Hit fist and use their energy to create things like energy mines. If you have the energy, battle smarts to think of such a thing, the right manipulation of energy and can fall through with creating something useful with your energy then it all comes down to thought and action.

For example Krillin could possibly make a ki sword, create energy mines and ranged energy punch attacks(assasin's fist) but without proper training he couldn't. Without proper battle smarts he wouldn't think to do such a thing. Without the right manipulation of energy his energy wouldn't withstand and implode due to his lack of mastery with the energy manipulation skill.

I just feel like armored cosmetic things like this should NEVER be limited seeing it's ways that each race can do such a thing, granted they have the requirements to do it.

This would potentially cause people to gravitate to one race if all the glorified skills can only be learn by 'said' race. 

Basically, I don't think toriyama would limit the use of energy amongst other races UNLESS legit reasons.. and it just seems like emelees are being placed by who had used them instead of who could potentially learn these techniques.
